annotator-core/src/main/java/edu/ucr/cs/riple/core/Annotator.java

```diff
@@ -314,5 +331,21 @@ private void forceResolveRemainingErrors() {
     injector.injectAnnotations(initializationSuppressWarningsAnnotations);
     // Update log.
     config.log.updateInjectedAnnotations(initializationSuppressWarningsAnnotations);
+    // Collect @NullUnmarked annotations on classes for any remaining error.
+    Utility.buildTarget(config);
+    remainingErrors =
+        Utility.readErrorsFromOutputDirectory(config, config.target, fieldDeclarationStore);
+    nullUnMarkedAnnotations =
+        remainingErrors.stream()
+            .filter(error -> !error.getRegion().isInAnonymousClass())
```

Comment:
Not for this PR, but long term, in theory, this should try to find the containing method/initializer and add `@NullUnmarked` there, no? (For anonymous inner classes). It can get complicated with nested anonymous classes and the places they can appear in, so definitely not asking for that in this PR, just wondering if that's in the roadmap for `1.3.6` or *after* `1.3.6`.

Reply:
Yes exactly. But I think the plan is to implement that in after `1.3.6`.